orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -56,8 +56,16 @@ class MonitorKatoTask implements RetryableTask {
     }
 
     Map<String, ? extends Object> outputs = [:]
-    if (status == ExecutionStatus.SUCCEEDED && !stage.context.containsKey("deploy.server.groups")) {
-      outputs["deploy.server.groups"] = getServerGroupNames(katoTask)
+    if (status == ExecutionStatus.SUCCEEDED) {
+      def deployed = getDeployedNames(katoTask)
+      if (!stage.context.containsKey("deploy.server.groups")) {
+        outputs["deploy.server.groups"] = getServerGroupNames(katoTask)
+      }
+      if (!stage.context.containsKey("deploy.jobs") && deployed) {
```

Comment:
Can `stage.context` ever contain both `deploy.server.groups` and `deploy.jobs`? If not, this can be `else if`.